### ISSUE TITLE
Desktop Menu Overhaul

### ISSUE THREAD

**Issue by Kreolis** [overte-org/overte#274]
These are external links to screenshots of the menus in desktop mode.

<a href="https://mega.nz/folder/S0d2BSAR#O_BZdwB65R7q5dih6CWSKw" target="_blank">BAR MENU</a>

Only menus that are different form the bar menu are recorded.
<a href="https://mega.nz/folder/Wt81FYJa#UW9_Z0bd5CmbuCvasgGzuQ" target="_blank">TABLET MENU</a>

Lets discuss how and what we want to change

**Comment by Kreolis484** [comment]
Tablet:

- unify all menus to not use scrolling but pages instead (scrollings is not so nice in VR)
- make appearance the same
- popups or dropdowns?

**Comment by ksuprynowicz** [comment]
Maybe instead of pages we could allow scrolling with controller analog sticks, like for example VRchat does?

**Comment by Kreolis484** [comment]
That is also a option. But scrolling with laser is not good. But still I think paging might be better. I hate the scrolling vrchat also (its better but not good, because e.g. it does not remember the position)

**Comment by ksuprynowicz** [comment]
For some cases labelled pages (like tabs) would indeed make most sense. Unlabelled pages on the other hand would be extremely difficult to browse for me (so for example world list). I love browsing worlds in VRchat just by scrolling - I can just press downwards on controller and look though them this way. Woud it be possible to at least keep is as an option for such case?

**Comment by Kreolis484** [comment]
Agreed scrolling on worlds is fine, but please with the stick. However I really hate scrolling in settings menus.

**Comment by ksuprynowicz** [comment]
I think that for settings menus tabs with thematically sorted optiond would indeed be best. In some menus scrolling will be unavoidable though as a fallback, especially list of audio sources, which is normally short on Windows, but can get very long on Linux.

**Comment by AleziaKurdis** [comment]
Scrolling in vr is possible if you have a decent area to do it (not those thin scroll bar from desktop UI)
This is why there is a "Sidewalk" area in the place api and the more api.
But it has indeed it's limit.
Pager can be very annoying too on very long list. (remember the explorer app)

Smaller menu but more hirarchical maybe? 
another thing that has it's limits.

The scroll using the thumbstick... it work well with Vive desktop. it seems to be a good option.

**Comment by daleglass** [comment]
The menu API needs an overhaul before any reorganization is possible, because the current API uses the literal menu names and paths as the argument. This means that moving or renaming anything is going to break the current code.

**Comment by Kreolis484** [comment]
buttons for on and off states on settings instead of sliders:

![image](https://user-images.githubusercontent.com/58556201/206894983-8cd6c50b-13e3-4d2e-82b1-afdde7f05b71.png)

**Comment by Armored-Dragon** [comment]
I am closing this issue as the major issues initially brought up have since been popped out into their own issues. Most issues are listed in this project: https://github.com/orgs/overte-org/projects/8. Feel free to further discuss specific quality of life or other adjustments on their respective issues or make a new issue about a specific desired change!
- https://github.com/overte-org/overte/issues/455
